website/server/controllers/api-v3/user.js

```diff
@@ -1267,32 +1267,6 @@ api.userReroll = {
 };
 
 /**
-* @api {post} /api/v3/user/addPushDevice Add a push device to a user
```

Comment:
I mean even if a 3rd party used this route (and I don't see why they would) it would just have not done anything. Now they'll get a 404